spec/unit/outputs/elasticsearch_spec.rb

```diff
@@ -2,48 +2,18 @@
 
 describe "outputs/elasticsearch" do
   context "registration" do
-    it "should register" do
-      output = LogStash::Plugin.lookup("output", "elasticsearch").new("embedded" => "false", "protocol" => "transport", "manage_template" => "false")
-      # register will try to load jars and raise if it cannot find jars
-      expect {output.register}.to_not raise_error
-    end
-
     it "should fail to register when protocol => http, action => create_unless_exists" do
```

Comment:
if we decide to remove this option from the vocabulary of this plugin, we can remove this test as well. woohoo!